arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -461,7 +542,7 @@
         "tags": [
           "Queues"
         ],
-        "operationId": "Queues_ListByNamspace",
+        "operationId": "Queues_ListByNamspaces",
```

Comment:
_ListByNamespaces, these changes were suggested in Autorest validation, reverted these changes.